allure-web/src/helpers/arrow.js

```diff
@@ -0,0 +1,11 @@
+import {SafeString} from 'handlebars/runtime';
+import {values} from '../util/statuses';
+
+
+export default function(item) {
+    var status = values.slice().reverse().reduce((result, current) => {
+        return item && (item[current] || item.status === current) ? current : result;
+    });
```

Comment:
reversed values:
'unknown', 'skipped', 'passed', 'broken', 'failed'
So init value for reduce is 'unknown'
1) for 'undefined' instead item - result is initial item - 'unknown'
2) for object with key 'status' - result will be 'value' of key 'status'
3) finally if item is statistics object, we take last status with non zero value in reversed 'values' order. For you example it is failed

We play with colored arrows for test groups, but decide that it is hard to understand.
Now this helper use only 'status' key. 

I fix it in next commit